### ISSUE TITLE
Flick with v6 from archlinux

### ISSUE THREAD

**Issue by tmpm697** [yshui/compton#140]
I use compton without any issue with ver 5.1.1 from `comunity/compton`, just update to v6 minutes ago and after kill compton process and refresh x11, compton seems buggy, it takes significant time for startx to start (include `compton &` in `.xinitrc`), I think it dues to time spending on starting `compton`. There's also a fast flick of black bar at top of my screen (I use nitrogen to set wallpapper).

**Comment by yshui** [comment]
`compton &` starts compton in backgroud, so that shouldn't have an effect on Xorg's startup time.

As for the black bar. Is it persistent, or is it just one flicker then gone? If it's persistent, can you take a screenshot/photo of it?

**Comment by WernerFP** [comment]
I can confirm autostart issues with Nitrogen and Compton v6 in Arch Linux with Openbox.
For me it works again as expected when I run nitrogen last (a little later than compton) - `~/.config/openbox/autostart`:
```
compton &
sleep 1 && nitrogen --restore &
```

**Comment by tmpm697** [comment]
Black bar is not persistent, @WernerFP I have `nitrogen` before `compton` so that might the problem, but it shouldn't be a problem.

**Comment by yshui** [comment]
I can't seem to reproduce the problem (cannot get the flicker, startup time looks fine to me).

If you want to investigate yourself, I'll help as much as I can. Otherwise I'm afraid there's not much I could do.

**Comment by dikiaap** [comment]
I get flickering only after booting. The flickering seems gone if I start apps like rofi in the example below

```sh
compton --diagnostics
**Version:** v6

### Extensions:

* Shape: Yes
* XRandR: Yes
* Present: Present

### Misc:

* Use Overlay: Yes
* Config file used: /home/dikiaap/.config/compton/compton.conf
```
 [compton.conf](https://ghostbin.com/paste/gyk79)

`exec --no-startup-id compton`

![x](https://user-images.githubusercontent.com/8542502/54950783-a3895d80-4f74-11e9-81e0-9a7ee3257959.png)

**Comment by kurnevsky** [comment]
Not sure if it's the same thing, but I have weird glitches after closing feh/mpv with version 6. The best way to reproduce is to try to open an image from terminal with `feh -F -Z` and close it several times. The window that gets a focus after closing feh goes into the state when it blinks every 1 second, becoming transparent or drawing the closed image. Here is my config: https://github.com/kurnevsky/dotfiles/blob/master/.config/compton.conf I also use xmonad and have Intel video card, if this matters.

**Comment by yshui** [comment]
@kurnevsky Does `stable/6` branch fix this problem?

**Comment by kurnevsky** [comment]
@yshui yes, thanks.

**Comment by dikiaap** [comment]
I have updated Compton to v6.2. This problem still exists.

**Comment by yshui** [comment]
@dikiaap You problem _seems_ different. Do you mind open another issue and you describe your problem in more detail? (preferrably include your hardware info, and a video clip demostrating your problem)

Thanks.
